runtime/types.ts

```diff
@@ -97,7 +97,7 @@ export type FormulaSpecification =
   | PropertyAutocompleteFormulaSpecification
   | GetPermissionsFormulaSpecification;
 
-export type ChainedCommandFormulaSpecification = GetPermissionsFormulaSpecification;
+export type ChainedCommandFormulaSpecification = GetPermissionsFormulaSpecification | SyncFormulaSpecification;
```

Comment:
Oh sorry I should have caught this last time, can you move this to somewhere under `testing/...`? Stuff in `runtime` is actually used in production, so we should keep the simulation stuff out of there.